app/models/distribution.rb

```diff
@@ -6,7 +6,8 @@ class Distribution < BaseModel
   belongs_to :audio_version_template
```

Comment:
Is there a case for switching this to be `audio_version_templates.first` right now?  So we won't need another CMS change to run the schema changes dropping this?

Reply:
it might work, but it's a relation which has more than a few methods and ways of being accessed. I'll try something naive, but it may not work for anything but read-only and updating the id.